Source/WTF/wtf/text/StringView.h

```diff
@@ -521,11 +521,13 @@ class StringView::UpconvertedCharactersWithSize {
     WTF_MAKE_FAST_ALLOCATED;
 public:
     explicit UpconvertedCharactersWithSize(StringView);
-    operator const UChar*() const { return m_characters; }
-    const UChar* get() const { return m_characters; }
+    operator const UChar*() const { return m_characters.data(); }
+    const UChar* get() const { return m_characters.data(); }
+    std::span<const UChar> span() const { return m_characters; }
```

Comment:
We could also add an `operator std::span<const UChar>` for the same reason we added an `operator const UChar*`. This would allow us to omit the `span()` at most call sites. That’s sort of the "trick" of this class’s original design. We can sort of pretend it is just returning a pointer to characters that already exist because the lifetime comes out correctly when you call `upconvertedCharacters` in a function argument expression.

Or you can leave it out. We don’t need to be tricky, I guess.